gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -184,7 +282,23 @@ def get_joint_freq_and_faf(
         },
     )
 
-    ht = ht.annotate(joint_freq=freq)
+    ht = ht.annotate(
+        joint_freq=freq,
+        **{
+            f"joint_{hist_struct}": hl.struct(
+                **{
+                    h: merge_histograms(
+                        [
+                            ht[f"exomes_{hist_struct}"][h],
+                            ht[f"genomes_{hist_struct}"][h],
+                        ]
+                    )
+                    for h in ht[f"genomes_{hist_struct}"]
```

Comment:
When you use hist_struct present from genomes to loop through, they are fully overlapping with the exomes ones, right? So it doesn't matter which one you use?

Reply:
It was a bit confusing to me at first because I was thinking about other histograms that we didn't calculate for genomes.